perseo-main/src/main/resources/perseo-core.properties

```diff
@@ -25,4 +25,5 @@ action.url = http://127.0.0.1:9090/actions/do
 
```

Comment:
After the last changes, I understand the changes in this file should be reverted, as `properties.path` and `action.path` are no longer used in the code.

Reply:
Fixed in b6d7148bbc37f5c8a5de01bf50c6c12a6d80d237